programmers/모의고사.js

```diff
@@ -0,0 +1,29 @@
+function solution(answers) {
+   const numberOne = [1,2,3,4,5];
+   const numberTwo = [2,1,2,3,2,4,2,5];
+   const numberThree = [3,3,1,1,2,2,4,4,5,5];
+   let firstCnt = 0;
+   let secondCnt = 0;
+   let thirdCnt = 0;
```

Comment:
찍기 방식을 하나의 patterns로 묶었고, count도 하나의 배열로 묶음
```suggestion
   const patterns = [
       [1, 2, 3, 4, 5],
       [2, 1, 2, 3, 2, 4, 2, 5],
       [3, 3, 1, 1, 2, 2, 4, 4, 5, 5]
   ];
   const count = [0, 0, 0];
   const result = [];
```